backend/src/services/users-to-groups/users-to-groups.service.ts

```diff
@@ -24,6 +24,50 @@ class UsersToGroups {
       userId: model.userId,
     };
   }
+
+  async getUsersByGroupId(
+    groupId: number,
+  ): Promise<UsersToGroupsResponseDto[]> {
+    const users = await this.#usersToGroupsRepository.getByGroupId(groupId);
+
+    return users;
+  }
+
+  async updateUsersByGroupId(usersToGroups: {
+    groupId: number;
+    userIds: number[];
+  }): Promise<void> {
+    const { groupId, userIds } = usersToGroups;
+    const groupUsers = await this.getUsersByGroupId(groupId);
+    const dbUsersId = groupUsers.map((user) => user.userId);
+
+    await Promise.all(
+      userIds.map((userId: number) => {
+        if (!dbUsersId.includes(userId)) {
+          return this.createUsersToGroups({
+            groupId,
+            userId,
+          });
+        }
+      }),
```

Comment:
Same thing about splitting as it was mentioned in comment for groups-to-permissions service